<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuStorage_Credentials">
     <property name="title">
      <string>Storage Credentials</string>
     </property>
     <widget class="QMenu" name="menuTXT_Storage">
      <property name="title">
       <string>TXT Storage</string>
      </property>
      <addaction name="actionCreate_Storage"/>
      <addaction name="separator"/>
      <addaction name="actionAdd_Site"/>
      <addaction name="actionGet_Password"/>
     </widget>
     <addaction name="menuTXT_Storage"/>
    </widget>
    <addaction name="menuStorage_Credentials"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Site">
   <property name="text">
    <string>Add Site</string>
   </property>
  </action>
  <action name="actionGet_Password">
   <property name="text">
    <string>Get Password</string>
   </property>
  </action>
  <action name="actionCreate_Storage">
   <property name="text">
    <string>Create Storage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
